lib/html2markdown.ex

```diff
@@ -3,9 +3,9 @@ defmodule Html2Markdown do
   A library for converting HTML to Markdown syntax in Elixir
   """
 
-  @navigation_classes ["footer", "menu", "nav", "sidebar", "aside"]
+  @default_navigation_classes ["footer", "menu", "nav", "sidebar", "aside"]
 
-  @non_content_tags [
```

Comment:
This is pretty aggressive in omitting html from further processing, so it seems reasonable to retain the default behavior, while allowing for context-specific configuration.